enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationOtpService.java

```diff
@@ -73,6 +76,8 @@ public class IdentityVerificationOtpService {
 
     private final AuditService auditService;
 
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Lets review what `ObjectMapper` instances we use.

Reply:
I agree, this was quick and working fix.